example/cavity/Fluid/system/controlDict

```diff
@@ -0,0 +1,77 @@
+/*--------------------------------*- C++ -*----------------------------------*\
+| =========                 |                                                 |
+| \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox           |
+|  \\    /   O peration     | Version:  v1912                                 |
+|   \\  /    A nd           | Website:  www.openfoam.com                      |
+|    \\/     M anipulation  |                                                 |
+\*---------------------------------------------------------------------------*/
+FoamFile
+{
+    version     2.0;
+    format      ascii;
+    class       dictionary;
+    location    "system";
+    object      controlDict;
+}
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+
+application         pimpleFoam;
+
+startFrom           latestTime;
+
+startTime           0;
+
+stopAt              endTime;
+
+endTime             100;
+
+deltaT              0.05;
+
+writeControl        runTime;
+
+writeInterval       0.10;
+
+purgeWrite          0;
+
+writeFormat         binary;
+
+writePrecision      14;
+
+writeCompression    off;
+
+timeFormat          general;
+
+timePrecision       9;
+
+runTimeModifiable   no;
+
+adjustTimeStep      no;
+
+maxCo               0.2;
```

Comment:
Prescribing a `maxCo` while setting `adjustTimeStep no` does not have any effect. Either enable dynamic time stepping, or remove the `maxCo`. A fixed time stepping is probably easier for the user to follow.